mslice/plotting/plot_window/plot_window.ui

```diff
@@ -84,6 +84,8 @@
    <addaction name="actionDataCursor"/>
    <addaction name="actionPlotOptions"/>
    <addaction name="actionToggleLegends"/>
+   <addaction name="actionInteractive_Cuts"/>
```

Comment:
I think we're running into space issues now with the toolbar. In my version the `Keep`/`Make Current` buttons are now hidden away and I have to click the `>>` to make them visible. Also, on the cut preview the `Save Cut` button is again hidden but by quite a few buttons that are disabled and off no use. Is it possible to remove the disabled buttons at least?

@mducle Any thoughts? Icons are one option but there has been negative reaction to these in the past. We could at least shorten things like "Save Plot"/"Print Plot" to Save/Print. Also, we have the data cursor button but it seems a bit of a waste of the space - should we just have the coordinates activated all of the time?

Reply:
@martyngigg I like the icons, but I'll ask around to see who really doesn't like them. 

I agree that we can remove the data cursor button and just have the coordinates activated all the time.

Reply:
@martyngigg @MatthewBowles So I think we can replace the "standard" buttons (e.g. what's in the equivalent window in Mantid 4.0) with icons, but leave the non-standard buttons (e.g. `Keep`, `Make Current`) as text. The objection seems to be that in some cases the functionality of an iconed-button is unguessable from the icon picture... But I think we can assume that people will know what `Save`, `Print`, `Zoom`, etc. means.

If we have the same icon set as the Mantid 4.0 plot windows, it would also make the "look" of both programs more uniform - something which was an issue discussed at the meeting iirc...

Reply:
@martyngigg Where can I get the icons used in Mantid?

Reply:
@MatthewBowles I think Mantid 4 uses [qtawesome](https://github.com/spyder-ide/qtawesome) with [Font Awesome](https://fontawesome.com/icons?m=free) icons. The relevant code is here:

https://github.com/mantidproject/mantid/blob/master/qt/applications/workbench/workbench/plotting/toolbar.py

Reply:
@mducle You're correct. Again though `qtawesome` is currently on in the Windows nightly build so for now we should only include that on the `new_gui` branch?

Reply:
Yes. That makes sense (to have it only in `new_gui`)